### DISCUSSION TITLE
GitHub  从 Profile/README 到 QQ空间

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#56] [1️⃣ 新鲜事]
![d876fa6746d33c539b4837833470daf](https://user-images.githubusercontent.com/15010826/170612740-b4bcda29-31c3-488e-9564-c66191e0391d.jpg)

- [Beautiful and attractive github profile](https://github.com/rzashakeri/beautify-github-profile)
- [6 Easy Steps to Create a Beautiful GitHub Profile README](https://sarah-hart-landolt.medium.com/6-easy-steps-to-create-a-beautiful-github-profile-readme-edc7840b2c7)
- [Awesome GitHub Profile README](https://github.com/abhisheknaiidu/awesome-github-profile-readme)
- [Awesome GitHub Profile READMEs](https://zzetao.github.io/awesome-github-profile/)

**Comment by zhicheng-ning** [comment]
> * 6 Easy Steps to Create a Beautiful GitHub Profile README

这个链接我这边貌似无法访问

**Reply by will-ww** [reply]
> > * 6 Easy Steps to Create a Beautiful GitHub Profile README
> 
> 这个链接我这边貌似无法访问

是吗？我可以耶~（KXSW后）

**Reply by zhicheng-ning** [reply]
ok~

**Comment by tyn1998** [comment]
还有这种用Actions把自己博客内容自动同步到profile的：https://simonwillison.net/2020/Jul/10/self-updating-profile-readme/

![image](https://user-images.githubusercontent.com/32434520/170614343-42ed21f2-2201-4069-8561-34dbf3a85cf0.png)

**Comment by sunny0826** [comment]
玩法挺多的，但是要适度 😸

**Reply by will-ww** [reply]
> 玩法挺多的，但是要适度 😸

哈哈，是的是的~

**Comment by xgdyp** [comment]
我觉得profile-Readme很适合GitHub的第一次实践，在Datawhale的git课程中，我把这个作为实操github的一个作业，效果还是很好的。我觉得满足了很多特征（学习成本低、见效快、好用）
